<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TLSWarningForm</class>
 <widget class="QDialog" name="TLSWarningForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TLS certificate is untrusted</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="pixmap">
      <pixmap resource="icons.qrc">:/icons/images/tls_warning.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="4">
    <widget class="QLabel" name="labelMessage">
     <property name="text">
      <string notr="true">Here comes the TLS error message text. It is multiline.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="QPlainTextEdit" name="textCertificate">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>170</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushAccept">
     <property name="text">
      <string>Accept</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="pushAcceptAlways">
     <property name="text">
      <string>Accept always</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="pushReject">
     <property name="text">
      <string>Reject</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
